src/query/spec.ts

```diff
@@ -28,11 +28,17 @@ export interface SpecQuery {
   /**
    * Array of encoding query mappings.
    * Note: Vega-Lite's `encoding` is an object whose keys are unique encoding channels.
-   * However, for CompassQL, the `channel` property of encoding query mappings can be wildcards. 
+   * However, for CompassQL, the `channel` property of encoding query mappings can be wildcards.
    * Thus the `encoding` object in Vega-Lite is flatten as the `encodings` array in CompassQL.
    */
   encodings: EncodingQuery[];
 
+  /**
```

Comment:
Remove the JSDoc comment or change it to normal comment (`// width and height//`? 

Note that this comment would be only applied to height when you hover over width variable. So if we have descriptive comment, better make them separate for width / height.  However, if the `width` property's JSDocs is simply "width", there is no point adding it.

Reply:
I think a good JSDocs note for these two variables are that `width` cannot be wildcard (yet).  Same for `height`.  

(I don't think we need to implement them as wildcard yet as there is no required use case.)

Reply:
done!